### PR TITLE
Disable DNS tests as they fail with CI

### PR DESCRIPTION
**Pull request by caglar10ur**

### CODE REVIEW

**Comment by mhagen-vmware** [comment]
lgtm

File: lib/dns/dns_test.go [hunk 1]
```diff
@@ -80,6 +82,8 @@ func TestForwarding(t *testing.T) {
 }
 
 func TestVIC(t *testing.T) {
+	t.Skipf("Failing with CI")
```
Comment (hickeng) [inline lib/dns/dns_test.go]: can you instead check the error from `bind` and report that - we cannot currently tell if this is a cascade failure from prior test (cannot reuse port?) or a separate problem.

**Comment by hickeng** [comment]
LGTM - please raise a second PR immediately with these re-enabled so we can iterate on a fix for them

**Comment by mdubya66** [comment]
Did an issue get raised to re-enable?

**Comment by caglar10ur** [comment]
https://github.com/vmware/vic/issues/1799

**Pull request merged**